security/org-account-context/main.tf

```diff
@@ -276,3 +276,368 @@ module "github_oidc_provider" {
+                value = "60days"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      plan_name = "180-days-retention"
+      rules = [
+        {
+          name                     = "monthly-180-days-retention"
+          schedule                 = "cron(0 3 L * ? *)"
+          enable_continuous_backup = false
+          lifecycle = {
+            delete_after = 180
+            cold_storage_after = 90 # Cold storage only stores full backups
+          }
+        }
+      ]
+      selections = [
+        {
+          name      = "select-prod-180-days"
+          resources = [
+            "arn:aws:rds:*:*:db:*",
+            "arn:aws:dynamodb:*:*:table/*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "prod"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
+                value = "180days"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      plan_name = "1-year-retention"
+      rules = [
+        {
+          name                     = "monthly-1-year-retention"
+          schedule                 = "cron(0 3 L * ? *)"
+          enable_continuous_backup = false
+          lifecycle = {
+            delete_after = 365
+            cold_storage_after = 90
+          }
+        }
+      ]
+      selections = [
+        {
+          name      = "select-prod-1-year"
+          resources = [
+            "arn:aws:rds:*:*:db:*",
+            "arn:aws:dynamodb:*:*:table/*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
+                value = "prod"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup"
+                value = "true"
+              },
+              {
+                key   = "aws:ResourceTag/dfds.data.backup_retention"
+                value = "1year"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      plan_name = "10-years-retention"
+      rules = [
+        {
+          name                     = "monthly-10-years-retention"
+          schedule                 = "cron(0 3 L * ? *)"
+          enable_continuous_backup = false
+          lifecycle = {
+            delete_after = 3650
+            cold_storage_after = 90
+          }
+        }
+      ]
+      selections = [
+        {
+          name      = "select-10-years"
+          resources = [
+            "arn:aws:rds:*:*:db:*",
+            "arn:aws:dynamodb:*:*:table/*"
+          ]
+          conditions = {
+            string_equals = [
+              {
+                key   = "aws:ResourceTag/dfds.env"
```

Comment:
The ResourceTag should be a variable since infrastructure-module is an open source project, and I guess any external user would prefer to specify their own ResourceTag name,